cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,56 @@
-describe('Cypress application', () => {
-  before(() => {
+/// <reference types="cypress" />
 
+describe('Cypress application', () => {
+  beforeEach(() => {
+    cy.visit('/alerts');
   });
 
   it('should have the ability to assert automatically resolved alerts', () => {
-
+    cy.get('#alertButton')
+      .click();
+    cy.on('window:alert', (text) => {
+      expect(text).to.equal('You clicked a button');
+    });
   });
 
-  it('should have the ability to assert scheduled allert', () => {
-
+  it('should have the ability to assert scheduled alert', () => {
+    cy.clock();
+    cy.window().then((win) => {
+      cy.stub(win, 'alert').as('windowAlert');
+      cy.get('#timerAlertButton').click();
+      cy.tick(5000).then(() => {
```

Comment:
Better to use cy.wait(5000) intead of cy.tick(5000).  cy.tick() is used to manage timeouts in tests, while cy.wait() is used to wait for events or responses in the test environment .

Reply:
noted